packages/commerce-sdk-react/src/hooks/ShopperBaskets/mutation.test.ts

```diff
@@ -193,23 +193,23 @@ describe('ShopperBaskets mutations', () => {
         '`%s` returns data on success',
         async (mutationName, options) => {
             mockMutationEndpoints(basketsEndpoint, oldBasket)
-            const {result, waitForValueToChange: wait} = renderHookWithProviders(() => {
+            const {result} = renderHookWithProviders(() => {
                 return useShopperBasketsMutation(mutationName)
             })
             expect(result.current.data).toBeUndefined()
             act(() => result.current.mutate(options))
-            await waitAndExpectSuccess(wait, () => result.current)
+            await waitAndExpectSuccess(waitFor, () => result.current)
```

Comment:
Unfortunately, we can't simply swap it like this because the APIs differ.

In the new API, `waitFor` expects something that will throw an error. So typically you'd call `expect`. See this great migration guide for more details: https://github.com/testing-library/react-hooks-testing-library/blob/chore/migration-guide/MIGRATION_GUIDE.md#waitforvaluetochange

Reply:
https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1184/files#diff-86d929c485f5fb0494d9b7c22841ae3d40858c74becaa2a02f6a3d6034872449R293 @joeluong-sfcc has a PR to fix these util funcs